app/src/docs/components/DocsPages/Products/DataUnions/index.jsx

```diff
@@ -3,13 +3,15 @@
 import React from 'react'
 import DocsHelmet from '$docs/components/DocsHelmet'
 import DocsLayout from '$docs/components/DocsLayout'
+import DataUnionsPlaceholderContent from '$docs/content/products/dataUnionsPlaceholder.mdx'
```

Comment:
Placeholder is the old "Data unions coming soon" text, we can write the new section with `DATA_UNIONS` enabled